VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -91,6 +96,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
             logger.error(ex);
         }
         loadData(getView());
+
+        String lastAccessedId = getArguments().getString(Router.EXTRA_LAST_ACCESSED_ID);
+        if (!TextUtils.isEmpty(lastAccessedId)) {
```

Comment:
This check is redundant, as the `updateRowSelection()` method also has the same check.